boards/arm/nrf9160_pca10090/board.cmake

```diff
@@ -0,0 +1,4 @@
+board_runner_args(nrfjprog "--nrf-family=NRF91")

```

Comment:
pin-reset is working for nrf9160_pca10090, so I modified this, @SebastianBoe , to align with the rest of the Nordic Dev-Kits

Reply:
Resolving this one (was only to make @SebastianBoe aware that the runner-args are now the default nrf board runner args)